xmtp_mls/src/storage/encrypted_store/identity_update.rs

```diff
@@ -85,6 +85,17 @@ impl DbConnection {
         })?)
     }
 
+    pub fn get_latest_sequence_id_for_inbox(&self, inbox_id: &str) -> Result<i64, StorageError> {
```

Comment:
@37ng I realized I gave you some bad advice last week on the sequence_id stuff.

Keeping the in-memory value up to date is a lot of hassle, so I switched to just using the latest value in the database.

Apologies for the churn.

Reply:
This makes total sense! I was wondering if we could just fetch sequence_id from Identity Updates table since it's the the source of truth, but that slipped off my radar.